src/components/status/status.stories.tsx

```
@@ -0,0 +1,407 @@
+    <Status variant="success">
+      <StatusIndicator />
+      <StatusLabel>Online</StatusLabel>
+    </Status>
+  ),
+  play: async ({ canvas }) => {
+    const status = canvas.getByText("Online").closest('[data-slot="status"]');
+    await expect(status).toHaveAttribute("data-variant", "success");
+  }
+};
+
+export const Error: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)